ui/src/environments/environment-configuration-loader.service.ts

```diff
@@ -0,0 +1,59 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class EnvironmentConfigurationLoaderService {
```

Comment:
I would probably just call this the ConfigLoaderService, this name is super long and it indicates that this is completely separate from the environment file now.

Reply:
Makes sense to me, will do!